recipes/muscle/build.sh

```
@@ -1,9 +1,16 @@
 #!/bin/bash
 
-cd src/
-sed -i.bak -e 's/ -static//' Makefile
+mkdir -p ${PREFIX}/bin
+cd src
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)